chain-signatures/test-contracts/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mpc-test-contract"
-version = "0.1.0"
+version = "1.0.0"
```

Comment:
test-contract should not be updated for versions since it's never used in the node